lib/sort.js

```diff
@@ -2,12 +2,12 @@
 
 // Compare for array.sort with priority
 //   priority - <string[]>, with priority
+// Returns: <Function>, function(s1, s2)
```

Comment:
There is no point to duplicate the signature in place of description:
```suggestion
// Returns: <Function>, comparison function that can be passed to `Array#sort()`
```